### DISCUSSION TITLE
How does "Default Terminal" work?

### DISCUSSION THREAD

**Discussion by lucien-gh** [microsoft/terminal#15861] [Q&A] [answered]
Why do I directly double-click the cmd.exe file in the System32 directory, it will also open in Windows Terminal? How is it achieved? Ask for answers

> "I want to know how it works"

**Comment by zadjii-msft** [comment, marked as answer]
What you're seeing is the "Default Terminal" feature in Windows 11, which lets you change which application is used as the default terminal emulator for command-line applications. That was previously always "conhost.exe", but now the Windows Terminal can be used as the default emulator too. 

_How_ it works is... complicated... and I believe we're tracking writing those docs in #9462. It was originally spec'd here: #7414

The other comment you have in this thread, about the leading spaces, looks like it's the same thing we're discussing with you in #15836, so I'm gonna collapse that as off-topic.

**Reply by DHowett** [reply]
I've promoted this to a discussion thread and removed the unrelated commentary from it.
